api/schemas/input/subject.json

```diff
@@ -7,11 +7,15 @@
     "lastname":         { "type": "string", "maxLength": 64 },
     "firstname_hash":   { "type": ["string", "null"] },
     "lastname_hash":    { "type": ["string", "null"] },
+
+    "age":              { "type": ["number", "null"] },
     "sex":              { "enum": ["male", "female", "other", "unknown", null] },
+    "race":             { "enum": ["American Indian or Alaska Native", "Asian", "Native Hawaiian or Other Pacific Islander", "Black or African American", "White", "More Than One Race", "Unknown or Not Reported", null] },
+    "ethnicity":        { "enum": ["Not Hispanic or Latino", "Hispanic or Latino", "Unknown or Not Reported", null] },
```

Comment:
Out of curiosity, what drives these fields? Homegrown? BIDS? Some other standard?